mono/metadata/w32subset.h

```diff
@@ -178,3 +178,43 @@
 #define HAVE_API_SUPPORT_WIN32_GET_MODULE_HANDLE 0
 #endif
 #endif
+
+#ifndef HAVE_API_SUPPORT_WIN32_GET_THREAD_CONTEXT
```

Comment:
I don't think this part is needed. GetThreadContext is always supported.

Reply:
Was just put there for consistency, but agree, we could drop it.